index.html

```diff
@@ -181,8 +181,9 @@ <h2>
       </header>
       <form id="form-element" action="https://formspree.io/f/meqdoagj" method="POST">
         <input class="input-field" type="text" name="name" maxlength="30" placeholder="Full name (required)" required>
-        <input class="input-field" type="email" name="email" placeholder="Email address (required)" required>
+        <input class="input-field" id="emaile" type="email" name="email" placeholder="Email address (required)" required>
         <textarea id="text-area" class="input-field" name="message" maxlength="500" placeholder="Enter text here (required)" required></textarea>
+        <small class="error">Error</small>
         <button id="form-button" type="submit">Get in touch</button>
       </form>
```

Comment:
Great job @julzedz 🤝

- [ ] As the previous reviewer suggested, kindly make sure your form submits when the email is lowercase.
![email](https://user-images.githubusercontent.com/63917725/189394498-11dcf7a7-f59f-44a0-b73f-9aa483cbd5c6.PNG)